Project/Travel App/Travel App/ContentView.swift

```diff
@@ -5,31 +5,7 @@ struct ContentView: View {
     @State var status = UserDefaults.standard.value(forKey: "status") as? Bool ?? false
     
     var body: some View {
-        NavigationView{
-            VStack {
-                if self.status {
-                    //homescreen
-                }
-                else {
-                    ZStack{
-                        NavigationLink(destination: SignUp(show: self.$show), isActive: self.$show) {
-                            Text("")
-                        }
-                        .hidden()
-                        Login(show: self.$show)
-                    }
-                    .navigationBarTitle("")
-                    .navigationBarHidden(true)
-                    .navigationBarBackButtonHidden(true)
-                    .onAppear() {
-                        NotificationCenter.default.addObserver(forName: NSNotification.Name("status"), object: nil, queue: .main) { (_) in
-                            
-                            self.status = UserDefaults.standard.value(forKey: "status") as? Bool ?? false
-                        }
-                    }
-                }
-            }
-        }
```

Comment:
@harshagl2002 This needs to be changed back (this was some of the rest of the group's code)